hass_nabucasa/remote.py

```diff
@@ -21,6 +22,8 @@
 RENEW_IF_EXPIRES_DAYS = 25
 WARN_RENEW_FAILED_DAYS = 18
 
+IS_CLOUD_REQUEST = ContextVar("IS_CLOUD_REQUEST", default=False)
```

Comment:
Should this have been lower-cased? It is not a constant?
Dunno, I'm fine with this, its just on the HA end, we don't capitalize them.